### DISCUSSION TITLE
Allowed file transfer from master

### DISCUSSION THREAD

**Discussion by owlysk** [rustdesk/rustdesk#3071] [General]
First at all I want to say thank you for awesome job for this software. I was long time looking for something like rustdesk. Thank you.

When I tried your amazing app, I noticied this behaviour:

When I am connected to remote desktop, I cant click on rustdesk window on remote-side (I think this is security feature and it's OK). When I tried to access to file transfer, on remote-side shows dialog to allow file transfer. Again, I cant allow this from remote-side, because security feature?
**But if I press spacebar, I can allow it!** 
It's bug? Pressing ENTER, or mouse click doesn't allow this.

**How did you install `RustDesk`?**
Deb file from website

---

**RustDesk version and environment**
Version 1.1.9 , Linux Ubuntu 22.04.1 LTS, X11

**Comment by rustdesk** [comment]
We are rethinking about the logic of this part in the coming version.

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/issues/2680
